crates/cairo-lint-core/tests/test_files/performance/inefficient_while_comp

```diff
@@ -0,0 +1,25 @@
+//! > while loop exit condition less than
```

Comment:
what happens in a more complex case ? for example that

```cairo
fn main() {
    let mut a = 1_u32;
    while a < 10 && true {
        a += 1;
    }
}
```

Reply:
Good point, need to go deeper :)